go/libraries/events/collector.go

```diff
@@ -85,7 +112,89 @@ func (c *Collector) Close() []*eventsapi.ClientEvent {
 
 	c.wg.Wait()
 
-	interf := c.val.Load()
+	return c.events
+}
+
+type sendingThread struct {
+	logCtx  context.Context
+	cancelF func()
+
+	batchCh chan []*eventsapi.ClientEvent
+	unsent  []*eventsapi.ClientEvent
```

Comment:
Seems like this is actually a local var?

Reply:
Almost. Returned from `stop()` for any un-emitted things.